src/watchmaker/managers/base.py

```diff
@@ -242,23 +159,13 @@ def download_file(self, url, filename, sourceiss3bucket=False):
                           '    url      = {0}\n'
                           '    filename = {1}'.format(url, filename))
 
-    def call_process(self, cmd):
-        if not isinstance(cmd, list):
-            exceptionhandler('Command is not a list.\n{0}'.format(str(cmd)))
-        rsp = subprocess.call(cmd)
-
-        if rsp != 0:
-            exceptionhandler('Command failed.\n{0}'.format(str(cmd)))
-
+    @abc.abstractmethod
     def create_working_dir(self, basedir, prefix):
         """
-        Create a directory in `basedir` with a prefix of `prefix`.
 
-        Args:
```

Comment:
Let's do that in a new PR. We can merge this. I'd also like to switch the print statements to log functions...